modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -6,20 +6,20 @@ import sdpTransform from 'sdp-transform';
 export default class RTCSessionDescription {
```

Comment:
AFAIR we already discussed about that and concluded that, since it's not an standard method at all and libjitsi does not use it anywhere, there is no value in adding it.